include/backend/wayland.h

```diff
@@ -32,6 +32,7 @@ struct wlr_wl_backend {
 	struct wl_shm *shm;
 	struct wl_seat *seat;
 	struct wl_pointer *pointer;
+	struct wlr_wl_pointer *wlr_wl_pointer;
```

Comment:
Why do we store both a `wlr_wl_pointer` and a `wl_pointer`?

Reply:
I couldn't figure how to access the wlr_wl_pointer from the wl_pointer.
The opposite is certainly possible, I didn't spend the time to look, happy to refactor all the rest around wlr_wl_pointer if you think it makes a difference.

Reply:
If it only had one of these, it should make the code simpler because you don't have to worry about keeping them in sync, or people needing to choose which one to access depending on the situation which would be valuable.